macOS builds fails with --require-cloud and cannot find openssl libraries

While investigating #8854 I found that I could not build Netdata on my macOS (_which is my primary dev machine_):

It fails with:

```
checking if cloud functionality should be enabled... yes
configure: WARNING: OpenSSL required for agent-cloud-link but not found. Try installing 'libssl-dev' or 'openssl-devel'.
checking if libmosquitto static lib is present (and builds)... no
checking if libwebsockets static lib is present... yes
checking if netdata agent-cloud-link can be enabled... no
configure: error: User required agent-cloud-link but it can't be built!
 FAILED
```

I had to do this to get it to buidl:

```
  503  export PKG_CONFIG_PATH="/usr/local/opt/openssl@1.1/lib/pkgconfig"
  504  ./netdata-installer.sh --dont-wait --dont-start-it --disable-go --require-cloud --install /tmp/netdata
  505  brew info openssl
  506  export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
  507  export CPPFLAGS="-I/usr/local/opt/openssl@1.1/include"
  508  set | grep PKG
  509  ./netdata-installer.sh --dont-wait --dont-start-it --disable-go --require-cloud --install /tmp/netdata
  510  history | tail
```

I could have sworn I fixed this by adding a commit to @Ferroin 's PR #8286 but I can no longer find the commits I added :/

--- a/netdata-installer.sh
+++ b/netdata-installer.sh
@@ -29,6 +29,7 @@ fi
 # Pull in OpenSSL properly if on macOS
 if [ "$(uname -s)" = 'Darwin' ] && [ -d /usr/local/opt/openssl/include ]; then
   export C_INCLUDE_PATH="/usr/local/opt/openssl/include"
+  export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
 fi
 
 # -----------------------------------------------------------------------------

